code/modules/surgery/bodyparts/head.dm

```diff
@@ -260,6 +340,47 @@
 			if(eyes.eye_color)
 				eyes_overlay.color = eyes.eye_color
 
+	else
+		if(!facial_hair_hidden && facial_overlay && (FACEHAIR in species_flags_list))
+			facial_overlay.alpha = hair_alpha
+			. += facial_overlay
+			if(facial_gradient_overlay)
+				. += facial_gradient_overlay
+
+		if(show_debrained)
+			. += mutable_appearance('icons/mob/human_face.dmi', "debrained", HAIR_LAYER)
+
+		else if(!hair_hidden && hair_overlay && (HAIR in species_flags_list))
+			hair_overlay.alpha = hair_alpha
+			. += hair_overlay
+			if(hair_gradient_overlay)
+				. += hair_gradient_overlay
+
+/mob/proc/set_haircolor()
```

Comment:
1. Auto-doc this
2. Include all parameters that you use in subtypes here, for reasons you will soon find out😄